bots/tests-policy

```diff
@@ -0,0 +1,360 @@
+    # failures that otherwise cause flakiness and false positives.
+    #
+    # The things we check here must:
+    #  * have no impact on users of Cockpit in the real world
+    #  * be things we tried to resolve in other ways. This is a last resort
+    #
+
+    trace = normalize_traceback(trace)
+
+    # HACK: An issue in phantomjs and QtWebkit
+    # http://stackoverflow.com/questions/35337304/qnetworkreply-network-access-is-disabled-in-qwebview
+    # https://github.com/owncloud/client/issues/3600
+    # https://github.com/ariya/phantomjs/issues/14789
+    if "PhantomJS or driver broken" in trace:
+        return True
+
+    # HACK: A race issue in phantomjs that happens randomly
+    # https://github.com/ariya/phantomjs/issues/12750
+    if "Resource Error: Operation canceled" in trace:
+        return True
+
+    # HACK: Interacting with sshd during boot is not always predictable
+    # We're using an implementation detail of the server as our "way in" for testing.
+    # This often has to do with sshd being restarted for some reason
+    if "SSH master process exited with code: 255" in trace:
+        return True
+
+    # HACK: Intermittently the new libvirt machine won't get an IP address
+    # or SSH will completely fail to start. We've tried various approaches
+    # to minimize this, but it happens every 100,000 tests or so
+    if "Failure: Unable to reach machine " in trace:
+        return True
+
+    # HACK: For when the verify machine runs out of available processes
+    # We should retry this test process
+    if "self.pid = os.fork()\nOSError: [Errno 11] Resource temporarily unavailable" in trace:
+        return True
+
+    return False
+
+
+# -----------------------------------------------------------------------------
+# Known Issue Matching and Filing
+
+def normalize_traceback(trace):
+    # All file paths converted to basename
+    return re.sub(r'File "[^"]*/([^/"]+)"', 'File "\\1"', trace.strip())
+
+def list_directories(dirs):
+    result = [ ];
+    for d in dirs:
+        for f in os.listdir(d):
+            result.append(os.path.join(d, f))
+    return result
+
+def check_known_issue(api, trace, image):
+    directories =  [ ]
+    image_naughty = os.path.join(BOTS, "naughty", image)
+    if os.path.exists(image_naughty):
+        directories.append(image_naughty)
+    trace = normalize_traceback(trace)
+    number = 0
+    for naughty in list_directories(directories):
+        (prefix, unused, name) = os.path.basename(naughty).partition("-")
+        try:
+            n = int(prefix)
+        except:
+            continue
+        with open(naughty, "r") as fp:
+            match = "*" + normalize_traceback(fp.read()) + "*"
+        # Match as in a file name glob, albeit multi line, and account for literal pastes with '[]'
+        if fnmatch.fnmatchcase(trace, match) or fnmatch.fnmatchcase(trace, match.replace("[", "?")):
+            number = n
+    return number
+
+def redact_audit_variables(message):
```

Comment:
I know it's just copy&paste'd code, but this is far from "obviously bug-free".  `git grep audit bots/naughty/` shows that we don't actually have any naughty issues with such numbers, they all use `*` properly. Can we just drop this?

Reply:
We could try and drop it, but this is probably material for a follow up, so we don't change too many things. This is just a renamed file after all.